packages/building-blocks/src/dropdown/index.tsx

```diff
@@ -0,0 +1,132 @@
+import React, {
+  HTMLAttributes,
+  KeyboardEvent,
+  useCallback,
+  useEffect,
+  useRef,
+  useState
+} from "react";
+
+import { getNextSibling, getPreviousSibling } from "../utils/dom-query";
+import { DropdownButton, DropdownMenu, DropdownWrapper } from "./styled";
+
+export interface Props extends HTMLAttributes<HTMLElement> {
+  buttonStyle?: React.CSSProperties;
+  isList?: boolean;
+  label?: string;
+  listLabel?: string;
+  text?: JSX.Element | string;
+  nav?: boolean;
+  AlignMenuLeft?: boolean;
+}
+
+/**
+ * Renders a dropdown menu. By default, only a passed "text" is rendered. If clicked,
+ * a floating div is rendered below the "text" with list contents inside. Clicking anywhere
+ * outside the floating div will close the dropdown.
+ */
+export const Dropdown = ({
```

Comment:
Instead of `export const Dropdown`, remove the `export` qualifier, and add `export default Dropdown` at the end of the file. Update files that import this component accordingly. This is to address a build warning I saw reviewing #726.